README.md

```diff
@@ -0,0 +1,82 @@
+FAGC Discord Bot
+---
+
+## Contents
+- [FAGC Discord Bot](#fagc-discord-bot)
+- [Contents](#contents)
+- [Explanation](#explanation)
+	- [Key terms](#key-terms)
+	- [Data Types](#data-types)
+- [Installation](#installation)
+
+## Explanation
+
+[⬆️ Head back up](#contents)
+
+FAGC (Factorio Anti-Grief Community) is a community of people who try to work against griefers in Factorio.
+
+### Key terms
+
+[⬆️ Head back up](#contents)
+
+- Community: A community is a group of people that play Factorio together and use this API in a way to prevent griefers on their servers
```

Comment:
```suggestion
- Community: A community is a group of people that play Factorio together and use this API in a way to prevent griefing on their servers
```